apps/ledger-live-mobile/e2e/bridge/client.ts

```diff
@@ -19,6 +22,17 @@ let retryCount = 0;
 const maxRetries = 5; // Maximum number of retry attempts
 const retryDelay = 500; // Initial retry delay in milliseconds
 
+const getAllFeatureFlags = (appLanguage: string): Partial<{ [key in FeatureId]: boolean }> => {
```

Comment:
maybe it'd make sense to:
- retrieve feature flags only if value !== false (to avoid pollution)
- would it be possible to make this code shared between LLD & LLM?

Reply:
Sure, I removed disabled flags for report 
[QAA-90](https://ledgerhq.atlassian.net/browse/QAA-90) for making common code